menu.js

```diff
@@ -0,0 +1,17 @@
+const menuBars = document.querySelector('.menu-bars');
+const mobMenu = document.querySelector('#mob-menu');
+const mobCloseMenu = document.querySelector('#mob-close-menu');
+
+const mobileNav = document.querySelector('#mob-menu>ul>li>a');
+
+menuBars.addEventListener('click', () => {
+  mobMenu.style.display = 'flex';
+});
+
+mobCloseMenu.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
```

Comment:
Great work making the mobile menu disappear when a user clicks on the x icons. Also, you did a great job implementing the right path when a user clicked on the options. 
However, when the user clicks on the about and the contact options, the mobile menu is still showing on the top of the page. As you can see here: 


![ezgif com-gif-maker](https://user-images.githubusercontent.com/60292761/142731667-a2725327-ad1c-45e3-9820-a7ec71aee9d1.gif)


**Please make sure that the mobile menu will disappear when the user clicks(or taps) any option of the mobile menu.**